co-authors-plus.php

```diff
@@ -164,6 +174,59 @@ public function action_init() {
 
 	}
 
+	/**
+	 * Determine if block editor sidebar integration should be loaded.
+	 *
+	 * @param WP_Post|int|null $post Post ID or object, null to use global.
+	 * @return bool
+	 */
+	public function is_block_editor( $post = null ) {
+		// $editor_active = use_block_editor_for_post( $post );
+
+		// // If it's a new post, $post will be null, so we check
+		// // block editor support for the post type
+		// if ( ! isset( $post ) ) {
+		// 	global $typenow;
+		// 	$editor_active = use_block_editor_for_post_type( $typenow );
+		// }
+
+		// if ( ! $editor_active ) {
+		// 	return false;
+		// }
+
+		// return (bool) apply_filters( 'coauthors_block_editor_integration', self::SIDEBAR_PLUGIN_ENABLED );
+		$screen = get_current_screen();
+
+		return $screen->is_block_editor();
```

Comment:
Since `is_block_editor()` was introduced somewhat recently, [in WP 5.0](https://developer.wordpress.org/reference/classes/wp_screen/is_block_editor/), what do you think about checking if it exists `method_exists( $screen, 'is_block_editor' )` and defaulting to false if not?

Reply:
Thanks, @lschuyler! Updated (and removed that commented code!). The tests appear to be passing now too. Let me know if there's anything else.

Reply:
Thanks @laras126! I'll take a look this week.

Reply:
This looks great @laras126! The tests are all passing for me now too.